src/components/GuestScan.tsx

```diff
@@ -161,6 +163,15 @@ const GuestScan: React.VFC<Props> = ({ handleScan, page }) => {
                 </CardContent>
               </Card>
             )}
+
+            {/* Exhibition Full Alert */}
+            {isFull && (
+              <Card>
+                <CardContent className={classes.noPadding}>
+                  <Alert severity="warning">滞在人数の上限に達しました。</Alert>
```

Comment:
自動リロードであることをちゃんと書きたい気がするんですがどうですかね
```suggestion
                  <Alert severity="warning">滞在人数の上限に達しました。空きができるまでそのままでお待ちください</Alert>
```
とかするかスピナーを置くか…?

Reply:
スピナー置くのもちょっと鬱陶しい感じがするし、うーん
「現在展示内は満員です。」とかはどう